src/components/LandingPage/Hero/AiIcon.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+export const AiIcon = () => {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M11.9996 10.0963V16.4159C11.9979 16.9997 11.8784 17.577 11.6482 18.1135C11.4179 18.6499 11.0818 19.1343 10.6599 19.5377C10.2379 19.941 9.73888 20.2551 9.19265 20.4609C8.64642 20.6668 8.06425 20.7602 7.48104 20.7357C6.89782 20.7111 6.32557 20.5691 5.79858 20.318C5.27159 20.067 4.80071 19.7121 4.41418 19.2747C4.02764 18.8373 3.7334 18.3263 3.54909 17.7725C3.36479 17.2186 3.29421 16.6332 3.34159 16.0514C2.43902 15.0836 1.86914 13.1861 1.86914 11.6038C1.86914 9.44846 2.92771 7.35201 4.44314 6.52744C4.36104 5.7746 4.50644 5.01444 4.86058 4.34504C5.21471 3.67564 5.76134 3.12774 6.42991 2.77204C7.09847 2.41634 7.8583 2.26917 8.61133 2.34951C9.36435 2.42986 10.076 2.73403 10.6545 3.22279"
+        stroke="currentColor"
```

Comment:
you could have all the paths in a `<g>` element and have all the common attributes there. Or join everything into a single path using svgo (https://jakearchibald.github.io/svgomg/). But its just a minor thing because I love SVG's feel free to not change it if you don't feel like its. Its not bad or anything